src/main/java/quotes/RecentQuotes.java

```diff
@@ -1,12 +1,5 @@
 package quotes;
 
-import com.google.gson.Gson;
-
-import java.io.IOException;
-import java.io.Reader;
-import java.nio.file.Files;
-import java.nio.file.Paths;
-
 public class RecentQuotes {

```

Comment:
The reason your RecentQuotes.toString is not working is because you have parameters in the method signature. 
In order for java to know about it and call it, it needs to be in the exact form of `public String toString(){ ... };`